content/docs/getting-started.md

```diff
@@ -0,0 +1,101 @@
+---
+
+This page is an overview of the React documentation and related resources.
+
+- [Try React](#try-react)
+- [Learn React](#learn-react)
+- [Staying Informed](#staying-informed)
+- [Versioned Documentation](#versioned-documentation)
+- [Something Missing?](#something-missing)
+
+## Try React
+
+React is designed for gradual adoption, and you can use as little or as much React as you need. Whether you want to get a taste of React, add some interactivity to a simple HTML page, or start a complex React-powered app, the links in this section will help you get started.
+
+### Online Playgrounds
+
+If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s/new). If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
+
+### Add React to a Website
+
+You can [add React to an HTML page in one minute](/docs/add-react-to-a-website.html). You can then either gradually expand its presence, or keep it contained to a few dynamic widgets.
+
+### Create a New React App
+
+When starting a React project, [a simple HTML page with script tags](/docs/add-react-to-a-website.html) might still be the best option.
+
+As your application grows, you might want to consider a more integrated setup. There are [several recommended solutions](/docs/create-a-new-react-app.html) for creating JavaScript applications with React. Each of them can work with little to no configuration and lets you take full advantage of the rich React ecosystem.
+
+## Learn React
+
+People come to React from different backgrounds and with different learning styles. Whether you prefer a more theoretical or a more practical approach, we hope you'll find something useful in this section.
+
+Like any unfamiliar approach, React does have a learning curve. With practice and some patience, you will get the hang of it.
+
+### JavaScript Resources
+
+The React documentation assumes some familiarity with the JavaScript language. You don't have to be an expert, but it's harder to learn both React and JavaScript at the same time. We recommend going through [this JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level. It will likely take you between 30 minutes and an hour but will help you feel more confident learning React.
+
+Whenever you get confused by something in JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [javascript.info](http://javascript.info/) are great websites to check. There are also [community support forums](/community/support.html) where you can ask for help.
+
+### Main Concepts
```

Comment:
You recommend people start here first? I always imagined most people should use the tutorial as their first code intro. My instinct would be to swap this with the tutorial. Also as someone familiar with the docs I had no idea where this would link before clicking it, not sure if that is a bad thing.

Reply:
I'm kind of torn on this. Could do both ways. Let's try Tutorial first.

I'm a bit worried that Tutorial being a single page makes it seem like a bigger commitment.